### ISSUE TITLE
Create docker image to run mkdocs locally

### ISSUE THREAD

**Issue by Schnitzel** [k0rdent/kcm#351]
instead of needing to run python commands locally it would be nicer and easier for documentation editors to run a docker image that has mkdocs installed.

**Comment by p5ntangle** [comment]
We can setup a github action to publish the docs automatically to github pages, local dev check will still need to setup their environment. 
In general it makes more sense to move the documentation for Project 2A to its own repo

**Comment by PrashantR30** [comment]
> We can setup a github action to publish the docs automatically to github pages, local dev check will still need to setup their environment. In general it makes more sense to move the documentation for Project 2A to its own repo

I see https://github.com/Mirantis/project-2a-docs already exist. I'll use that one.

**Comment by p5ntangle** [comment]
@PrashantR30 Yes, thanks. It would be great if we could quickly work through build the basic TOC and start publishing docs.

**Comment by PrashantR30** [comment]
> [@PrashantR30](https://github.com/PrashantR30) Yes, thanks. It would be great if we could quickly work through build the basic TOC and start publishing docs.

Will be adding content as per https://github.com/Mirantis/hmc/issues/337 in another PR
